app/controllers/concerns/periodable.rb

```diff
@@ -0,0 +1,15 @@
+module Periodable
+  extend ActiveSupport::Concern
+
+  included do
+    before_action :set_period
+  end
+
+  private
+    # @return [void]
```

Comment:
We typically try to avoid YARD comments and would not need them here since this is a private method